README.md

```diff
@@ -6,21 +6,43 @@ This repository is used to track anything related to the platform that powers SU
 
 Short description of the workflow and of the columns in the board.
 
+### GitHub Milestones
+
+Mostly used for grouping for long standing projects such as KTLO work.
+
 ### Labels
 
-Each label in this board signifies a specific project.
+Labels are used to easily distinguish cards that have them attached. We should keep them to the minimum in order for the cards that hold them to easily stand out.
+
+There are two kind of labels:
+
+- Labels that correspond to a specific project (kitsune vs respond). There are as many labels as the projects that the team is working on.
+- Labels used to highlight an issue. This is a short list of labels.
+  - Bug
 
 ### Columns
 
+#### Triage column
+
+Catch-all column. Anything filed in this repository goes to this column.
+Triage should happen regularly and the column should not have many cards. A number of 20 should
+be the maximum.
+
+A card from here either moves to the backlog or it is closed.
+
+**A card can be closed, even if it is valid, if it is not going to be worked on in more than three months**.
```

Comment:
How does this interact with bugs we want to have open for community etc to work on? Perhaps we should transfer the issue into the project-specific repository?

Reply:
We can add a label for this type of cards. I will add it in the PR.